pkgcfg/package-configuration/nginx/nginx.conf

```diff
@@ -0,0 +1,20 @@
+events {}
+http{
+server {
+         listen       80;
+         listen       [::]:80;
```

Comment:
Why the two separate listen calls?

Reply:
Not sure, copy pasted this from @Lagoja 's template